temoa/data_processing/db_query.py

```
@@ -1,51 +1,85 @@
         cur.execute(query_string)
 
-        for row in cur:
-            db_result.append(row)
+        db_result.extend(cur)
 
         cur.close()
         con.close()
-        return 'Query Result: %s' % ''.join(db_result)
+        # Convert list of tuples to a more readable string representation
+        result_str = '\n'.join(map(str, db_result))
+        return f'Query Result:\n{result_str}'
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)